bindings/python/slimt.cpp

```diff
@@ -134,6 +140,11 @@ class PyService {
     return Service(config);
   }
 
+  static void change_ranges_to_utf8(Response &response) {
+    response.source.to(slimt::Encoding::UTF8);
+    response.target.to(slimt::Encoding::UTF8);
+  }
```

Comment:
In the current state, this function is private to Python (because we motivated this UTF8 notions by using Python). However, this part (converting a response to utf8 - which is simply relating the call to constitutent annotations) also finds use sometimes in CPP (example: [translateLocally](https://github.com/XapaJIaMnu/translateLocally/blob/a210037760ca3ca9016ca1831c97531318df70fe/src/Translation.cpp#L98), KDE.

So might as well take it out and expose in the C++ library, and allow Python to use.

Reply:
https://github.com/jerinphilip/slimt/pull/38/commits/a1064196d2bfae4eb709ce8aa7c53bef021fe618